### PR TITLE
Add default maven configuration

### PR DESCRIPTION
**Pull request by garagatyi**
@skabashnyuk @eivantsov Please take a look

### CODE REVIEW

File: .mvn/jvm.config [hunk 1]
```diff
@@ -0,0 +1 @@
+-Xmx2048m -Xms1024m -Djava.awt.headless=true -Djava.net.preferIPv4Stack=true -Dgwt.compiler.localWorkers=1
```
Comment (skabashnyuk) [inline .mvn/jvm.config]: new line

Comment (garagatyi) [inline .mvn/jvm.config]: added

**Comment by skabashnyuk** [comment]
ok

**Comment by vparfonov** [comment]
ok

**Pull request merged**